calibr8/utils.py

```diff
@@ -172,22 +133,24 @@ def plot_t_band(ax, independent, mu, scale, df, *, residual_type: typing.Optiona
     artists : list of matplotlib.Artist
         the created artists (1x Line2D, 6x PolyCollection (alternating plot & legend))
     """
+    assert hasattr(model.scipy_dist, "ppf"), "Only Scipy distributions with a ppf method can be used for the continuous likelihood bands."
```

Comment:
```suggestion
    if not hasattr(model.scipy_dist, "ppf"):
        raise ValueError("Only Scipy distributions with a ppf method can be used for the continuous likelihood bands.")
```
This way the coverage calculation is more accurate.
Also `assert`s could be disabled globally with `python -O`.